src/core/components/live-response.jsx

```diff
@@ -74,7 +74,7 @@ export default class LiveResponse extends React.Component {
     return (
       <div>
         { curlRequest && (requestSnippetsEnabled === true || requestSnippetsEnabled === "true"
-          ? <RequestSnippets request={ curlRequest }/>
+          ? <RequestSnippets getComponent={getComponent} request={ curlRequest }/>
```

Comment:
`RequestSnippets` is already access as container in https://github.com/swagger-api/swagger-ui/blob/7dfada89e1667d480828707f1ae56d6b18f03f72/src/core/components/live-response.jsx#L71. No need to pass `getComponent` prop.

Reply:
When `getComponent` is not passed as a prop I'm getting es-lint errors: 'getComponent' is not defined